daemon/SecretsImpl/secrets.cpp

```diff
@@ -72,6 +72,7 @@ void Daemon::ApiImpl::SecretsDBusObject::userInput(
     modifiedParams.setOperation(InteractionParameters::RequestUserData);
     modifiedParams.setCollectionName(QString());
     modifiedParams.setSecretName(QString());
+    modifiedParams.setPluginName(QString());
```

Comment:
Aren't these empty string setter calls unnecessary?

Reply:
Yes, this is the case where we are receiving the interaction parameters from the client application (e.g in case they are specifying echo mode, alphanumeric vs pin-entry, etc) - but we don't want them to set the collection/secret/plugin name (so we overwrite it with empty data).

This affects later on how we generate the prompt text, as when the Crypto side asks the Secrets side to perform the same sort of request (userInput) it will pre-fill the plugin+collection+secretName data, so that we generate the appropriate string, without including a "warning - data will be returned to the application" prefix, etc.

Reply:
I'll add a code comment here to clarify this.